package.json

```diff
@@ -10,13 +10,21 @@
   "repository": "github:openapi-js/json-schema-to-openapi-schema",
   "author": "Phil Sturgeon <me@philsturgeon.uk>",
   "license": "MIT",
+  "engines": {
+    "node": ">=8"
+  },
   "devDependencies": {
     "coveralls": "^3.0.0",
     "mocha": "^5.0.0",
+    "nock": "^11.3.6",
     "nyc": "^11.6.0",
     "should": "^13.2.0"
   },
   "dependencies": {
-    "@cloudflare/json-schema-walker": "^0.1.1"
+    "@cloudflare/json-schema-walker": "^0.1.1",
+    "@stoplight/json-ref-resolver": "^2.3.0",
+    "@stoplight/yaml": "^3.3.1",
+    "node-fetch": "^2.6.0",
+    "tslib": "^1.10.0"
```

Comment:
This is due to `json-ref-resolver`. Will add `tslib` to `json-ref-resolver` dependencies.